src/AWS.Deploy.Recipes/RecipeDefinitions/ConsoleAppECSFargateService.recipe

```diff
@@ -8,6 +8,7 @@
     "CdkProjectTemplate": "../CdkTemplates/ConsoleAppECSFargateService",
     "CdkProjectTemplateId": "netdeploy.ConsoleAppEcsFargateService",
     "Description": "Deploy a console application as a service to Amazon Elastic Container Service (Amazon ECS). Recommended for applications that can be deployed as a container image. If your project does not contain a Dockerfile, one will be generated for the project.",
+    "ShortDescription": "Deploy a console application as a service to Amazon Elastic Container Service (Amazon ECS).",
```

Comment:
Suggested Text:

"Console application deployed as a service to Amazon Elastic Container Service (Amazon ECS)."

Reply:
Fixed